src/go/org/hypertrace/agent/config/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/hypertrace/agent-config/src/go/org/hypertrace/agent/config
```

Comment:
can we not just use the top level go module for the config, as that's all this repo will be used for?